.golangci.yaml

```diff
@@ -119,23 +119,23 @@ linters:
     - gocyclo
     - gofmt
     - gofumpt
-    - golint
+    # - golint
```

Comment:
This is replaced by revive and revive is already enabled.

Reply:
More revive work is covered here https://github.com/RamenDR/ramen/pull/1162 but this is fine.